pennylane/qnodes/decorator.py

```diff
@@ -79,6 +79,9 @@ def QNode(func, device, *, interface="autograd", mutable=True, diff_method="best
 
             * ``None``: a non-differentiable QNode is returned.
 
+        h (float): step size for parameter shift or the finite
+            difference method
```

Comment:
```suggestion
        h (float): step size for the finite-difference method
```
Note that this currently doesn't change the step-size for the parameter shift method --- that will likely require more work in a separate PR, as changing the divisor in the parameter shift method also involves making a corresponding change to the shift itself (so `QubitDevice._pd_analytic` would need to be modified)

Reply:
Another thought: the default value of `h` should be specified here as well.

Reply:
Oh, right, thanks for getting that! :)

Would you suggest to include the default value of `h` here in the docstring or just in general in `decorator.py`?

Reply:
The default value should go in the docstring, whereever `h` is defined :)